packages/salesforcedx-vscode-aura/package.json

```diff
@@ -0,0 +1,104 @@
+{
+  "name": "salesforcedx-vscode-aura",
+  "displayName": "Lightning",
+  "description": "Provides code-editing features for Lightning",
+  "qna": "https://github.com/forcedotcom/salesforcedx-vscode/issues",
+  "bugs": {
+    "url": "https://github.com/forcedotcom/salesforcedx-vscode/issues"
+  },
+  "repository": {
+    "url": "https://github.com/forcedotcom/salesforcedx-vscode"
+  },
+  "icon": "images/VSCodeLWC.png",
+  "galleryBanner": {
+    "color": "#ECECEC",
+    "theme": "light"
+  },
+  "version": "45.6.0",
+  "publisher": "salesforce",
+  "preview": true,
```

Comment:
Why are we releasing it as preview ?

Reply:
Let's not do preview. This is miles ahead of the current version even.

Reply:
Yeah, I'll remove this when I merge with the lightning package